frontend/src/components/search/SearchResultsListFetch.tsx

```diff
@@ -15,20 +16,31 @@ export default async function SearchResultsListFetch({
   searchResultsPromise,
 }: ServerPageProps) {
   const searchResults = await searchResultsPromise;
-
-  const maxPaginationError = null;
   const t = await getTranslations("Search");
 
+  if (
```

Comment:
where this is being done is more or less arbitrary since the same promise is being awaited in multiple components. That seems weird, which is one of the reasons I did the refactor in https://github.com/HHS/simpler-grants-gov/pull/3796